packages/libsql/src/mod.ts

```diff
@@ -0,0 +1,70 @@
+import type { StorageAdapter } from './deps.deno.ts';
+import type { Client } from './deps.deno.ts';
+
+/**
+ * Storage adapter for Turso's libSQL.
+ */
+export class LibSQLAdapter<T> implements StorageAdapter<T> {
+  protected client: Client;
+  protected table: string;
+
+  private constructor(opts: { client: Client, table: string }) {
+    this.client = opts.client;
+    this.table = opts.table;
+  }
+
+  /**
+   * @param opts options
+   * @param opts.table - Name of table where data should be stored
+   * @param opts.client - Turos' libSQL client
+   * @returns A libSQL storage adapter
+   *
+   */
+  static async create<T>(opts: { client: Client, table: string }) {
+    const createTableStatement = `
+      CREATE TABLE IF NOT EXISTS "${opts.table}" (
+        key TEXT NOT NULL,
+        value TEXT
+      );`;
+    await opts.client.execute(createTableStatement);
+
+    const createIndexStatement = `CREATE UNIQUE INDEX IF NOT EXISTS "IDX_${opts.table}" ON "${opts.table}" (key);`;
+    await opts.client.execute(createIndexStatement);
+
+    return new LibSQLAdapter<T>(opts);
+  }
```

Comment:
Some users have correctly pointed out that `await create()` to be called on every application start is bad, because sometimes bots can be used in serverless environments where this call takes a decent amount of time. Instead, I'd like to instruct them in the readme that they should create a table in their database. Then we will accept the table name, client, as an option. And we wouldn't need a static `create` method

Thats also about `psql` adapter, which i didnt refactored yet.